packages/web-client/app/components/card-pay/create-merchant-workflow/prepaid-card-choice/index.ts

```diff
@@ -105,10 +104,8 @@ export default class CardPayCreateMerchantWorkflowPrepaidCardChoiceComponent ext
     if (this.prepaidCards.length) {
       this.prepaidCards.forEach((c) => {
         let isLowBal = c.spendFaceValue < this.merchantRegistrationFee;
-        let option: DropdownOption = {
-          card: c,
-          disabled: isLowBal,
-        };
+        let option: DropdownOption = c;
+        option.disabled = isLowBal;
```

Comment:
@Aierie this is ready for another review. This component is not meant to be specific to web-client. It will eventually evolve into a card picker for the card compiler. This is the initial design we have before any card compiler related changes.